eo-maven-plugin/src/test/java/org/eolang/maven/UnplaceMojoTest.java

```diff
@@ -123,19 +126,20 @@ void testKeepBinaries(@TempDir final Path temp) throws Exception {
             .with("keepBinaries", new SetOf<>("**foo5.class"))
             .execute();
         MatcherAssert.assertThat(
-            new Home().exists(foo),
+            Files.exists(foo),
             Matchers.is(true)
         );
         MatcherAssert.assertThat(
-            new Home().exists(Paths.get(String.valueOf(pparent))),
+            Files.exists(Paths.get(String.valueOf(pparent))),
             Matchers.is(true)
         );
     }
 
     @Test
     void testKeepRemoveBinaries(@TempDir final Path temp) throws Exception {
         final Path foo = temp.resolve("a/b/c/foo6.class");
-        new Home().save("testKeepRemoveBinaries", foo);
+        final Home home = new Home(temp);
```

Comment:
Can we inline this variable? https://www.yegor256.com/2015/09/01/redundant-variables-are-evil.html

Reply:
@volodya-lombrozo fixed